### PR TITLE
Fixed broken tabs on the about page

### PR DESCRIPTION
**Pull request by kiwiupover**
Before 
image

After
image

Thanks @trombom

### CODE REVIEW

File: source/javascripts/app/about/inline-examples.js [hunk 1]
```diff
@@ -55,12 +55,22 @@ function generateViewerApp($elem, files) {
...
+    },
 
     isSelected: function() {
-      return this.get('model') === this.get('controllers.application.selectedTab');
-    }.property('controllers.application.selectedTab')
+      return this.get('item') === this.get('selectedTab');
+    }.property('selectedTab', 'item'),
+
+    layout: Ember.Handlebars.compile(
+      '<a class="{{if isSelected \'active\'}}" href="#">{{item.name}}</a>'
```
Comment (mixonic) [inline source/javascripts/app/about/inline-examples.js]: Can this go into a `<script>` tag template?

Comment (kiwiupover) [inline source/javascripts/app/about/inline-examples.js]: Yea that is a good idea @mixonic

**Comment by kiwiupover** [comment]
@mixonic we are ready to roll again.

**Comment by mixonic** [comment]
Super :-)

**Pull request merged**